### DISCUSSION TITLE
Smooth scrolling/sync in 2D platform games?

### DISCUSSION THREAD

**Discussion by Tasosgemah** [86Box/86Box#3261] [Q&A] [answered]
Is there a way to sync the guest computer screen with the host monitor?

When i play some 2D games like Earthworm Jim on DosBox Pure, the scrolling is smooth as butter, just like the console versions. But on the 86box, even on a powerful enough Pentium guest, the same game is erratic, despite the game running at full speed. 

I suspect it's because the Guest system in 86box doesn't sync perfectly with the actual monitor. This isn't as noticeable with 3D games that have unstable frame rates themselves, but even then you still lose a few frames because of the loss of sync.

My monitor is 240hz, which should divide perfectly with the guest system's reported 60hz rate. The monitor also supports freesync/Gsync but i don't see 86box using it. Is there a way for 86Box to sync in the same way RetroArch/DosBox Pure can?

**Comment by ts-korhonen** [comment]
If you use hardware renderers (other than Qt Software) then g-sync will be used, at least on Windows. You need to have it set active on windowed mode too, not just fullscreen. I assume same is for freesync. Also, restart 86Box if you change renderers, it seems not to take effect when changed.

**Reply by Tasosgemah** [reply]
> If you use hardware renderers (other than Qt Software) then g-sync will be used, at least on Windows. You need to have it set active on windowed mode too, not just fullscreen. I assume same is for freesync. Also, restart 86Box if you change renderers, it seems not to take effect when changed.

Unfortunately, this didn't work. I think it might be even worse actually as the scrolling stutters very heavily. I followed the instructions (gsync/freesync was already active for windowed applications, selected QT Opengl and restarted the program before trying it.

**Reply by Kappa971** [reply, marked as answer]
See this: https://github.com/86Box/86Box/discussions/3340
